integrations/claude_query_server.py

```
@@ -0,0 +1,131 @@
+    raise
+
+from integrations.agency_bridge import (
+    get_index,
+    get_tool,
+    get_statistics,
+    get_system_prompt_text,
+    search_tools,
+    query_knowledge_base,
+)
+
+SERVER_PORT = int(os.environ.get("KNOWLEDGE_SERVER_PORT", 8200))
```

Ruff (0.15.9)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)